docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,19 +5,28 @@
    "id": "06bdcdfc",
```

Comment:
<p><code>DataDuplicates</code> check can also use a specific subset of columns (or alternatively use all columns except specific <code>ignore_columns</code> to check duplication):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think showing ignore_columns is important in the integrity checks.</p>